bin/inference/pycbc_inference_extract_samples

```diff
@@ -54,33 +52,11 @@ parser.add_argument("--posterior-only", action="store_true", default=False,
                          "all walkers in {samples_group/{param}/. Default is "
                          "for the copied file to have the same structure "
                          "as the input file.")
-# options for down selecting parameters
-parser.add_argument("--parameters", type=str, nargs="+",
-    metavar="PARAM[:NAME]",
-    help="Name of parameters to copy. If none provided will copy all of "
-         "the variable_args in the input file. If provided, the "
-         "parameters can be any of the variable args in "
-         "the input file, derived parameters from them, or any function "
-         "of them. Syntax for functions is python; any math functions in "
-         "the numpy libary may be used. Can optionally also specify a "
-         "name for each parameter, in which case the parameter will be "
-         "saved with the given name in the output file.")
-parser.add_argument("--thin-start", type=int, default=None,
-    help="Sample number to start collecting samples to plot. If none "
-         "provided, will start at the end of the burn-in.")
-parser.add_argument("--thin-interval", type=int,
-    default=None,
-    help="Interval to use for thinning samples. If none provided, will "
-         "use the auto-correlation length found in the file.")
-parser.add_argument("--thin-end", type=int, default=None,
-    help="Sample number to stop collecting samples to plot. If none "
-         "provided, will stop at the last sample from the sampler.")
-parser.add_argument("--iteration", type=int, default=None,
-    help="Only retrieve the given iteration. To load the last n-th sampe "
-         "use -n, e.g., -1 will load the last iteration. This overrides "
-         "the thin-start/interval/end options.")
```

Comment:
Ok, if you're going to use the inference results option group (which I agree makes sense), then you need to make a couple of changes to that code. First, that group will add the parameters-group option, which doesn't make sense for this program. Maybe add a keyword argument to `add_inference_results_option_group` to exclude that option. Second, change the help messages to remove mentions of plots, e.g., for the parameters option, change "Name of parameters to plot" to "Name of parameters to load."

Reply:
> First, that group will add the parameters-group option, which doesn't make sense for this program.

Its used in `option_utils.results_from_cli`. Its not needed for the HDF case.